tests/test_linearize.py

```diff
@@ -125,8 +125,8 @@ def setUp(self):
                                     [[1e-5, 1e-7], [1.1e-6, 1e-7], [2.1e-6, 1e-7]]], dtype=float)
 
         # Spline coefficients: should match a 1e-6 Squared solution
-        self.splineCoeffs = np.array([-100, 0.0, 1000, 2000, 3000, 4000, 5000,
-                                      0.0, 0.0, 1.0, 4.0, 9.0, 16.0, 25.0])
```

Comment:
Hmm, presumably this would have generated at least slightly negative values between -100 and 0 before, no? Did that cause any issues? But either way, starting it at zero definitely seems to make more sense.

Reply:
I think it was just poor thinking on my part when I wrote the test.  If that does match the `1e-6` squared solution, then that negative point gains flux from a correction entirely derived from the positive data, which doesn't make any sense at all.